lib/nautic_net/protobuf/data_set.proto

```diff
@@ -87,3 +87,9 @@ message TrackerSample {
   sint32 rssi = 1; // dBm
   RoverData rover_data = 2;
 }
+
+message AttitudeSample {
```

Comment:
The DataPoint definition also needs to be updated on line 39, so that the AttitudeSample can be put into the `sample` field.

```
  message DataPoint {
    // ...
    oneof sample {
      // ...
      AttitudeSample attitude = 23;
    }
  }
```

Reply:
Makes sense. Updated and also ran `./generate_ex.sh`